src/cts/src/TritonCTS.tcl

```diff
@@ -80,6 +80,7 @@ sta::define_cmd_args "clock_tree_synthesis" {[-wire_unit unit]
                                              [-sink_clustering_size] \
                                              [-sink_clustering_max_diameter] \
                                              [-sink_clustering_enable] \
+                                             [-max_fanout] \
```

Comment:
We want to honor SDC set_max_fanout constraint by default.   Let's add -ignore_max_fanout If user wants to ignore max fanout limit,

Reply:
That seems ultra confusing. If a user sets this flag and the SDC flag it seems like we could offer a warning, but principle of least surprise seems to be we should honor the more specific constraint in the CTS call.

Reply:
We discussed today not needing the flag.  Please wait for the next update.